test/components/bbox.dart

```diff
@@ -0,0 +1,97 @@
+import 'package:test/test.dart';
+import 'package:turf/bbox.dart';
+import 'package:turf/helpers.dart';
+
+main() {
+  final pt = Feature<Point>(geometry: Point(coordinates: Position.named(lat: 102.0, lng: 0.5)));
+  final line = Feature<LineString>(
+      geometry: LineString(coordinates: [
+    Position.named(lat: 102.0, lng: -10.0),
+    Position.named(lat: 103.0, lng: 1.0),
+    Position.named(lat: 104.0, lng: 0.0),
+    Position.named(lat: 130.0, lng: 4.0),
+  ]));
+  final poly = Feature<Polygon>(
+      geometry: Polygon(coordinates: [
+    [
+      Position.named(lat: 101.0, lng: 0.0),
+      Position.named(lat: 101.0, lng: 1.0),
+      Position.named(lat: 100.0, lng: 1.0),
+      Position.named(lat: 100.0, lng: 0.0),
+      Position.named(lat: 101.0, lng: 0.0),
+    ],
+  ]));
+  final multiLine = Feature<MultiLineString>(
+      geometry: MultiLineString(coordinates: [
+    [
+      Position.named(lat: 100.0, lng: 0.0),
+      Position.named(lat: 101.0, lng: 1.0),
+    ],
+    [
+      Position.named(lat: 102.0, lng: 2.0),
+      Position.named(lat: 103.0, lng: 3.0),
+    ],
+  ]));
+  final multiPoly = Feature<MultiPolygon>(
+      geometry: MultiPolygon(coordinates: [
+    [
+      [
+        Position.named(lat: 102.0, lng: 2.0),
+        Position.named(lat: 103.0, lng: 2.0),
+        Position.named(lat: 103.0, lng: 3.0),
+        Position.named(lat: 102.0, lng: 3.0),
+        Position.named(lat: 102.0, lng: 2.0),
+      ],
+    ],
+    [
+      [
+        Position.named(lat: 100.0, lng: 0.0),
+        Position.named(lat: 101.0, lng: 0.0),
+        Position.named(lat: 101.0, lng: 1.0),
+        Position.named(lat: 100.0, lng: 1.0),
+        Position.named(lat: 100.0, lng: 0.0),
+      ],
+      [
+        Position.named(lat: 100.2, lng: 0.2),
+        Position.named(lat: 100.8, lng: 0.2),
+        Position.named(lat: 100.8, lng: 0.8),
+        Position.named(lat: 100.2, lng: 0.8),
+        Position.named(lat: 100.2, lng: 0.2),
+      ],
+    ],
+  ]));
+  final fc = FeatureCollection(features: [pt, line, poly, multiLine, multiPoly]);
+
+  test("bbox", () {
+    // FeatureCollection
+    final fcBBox = bbox(fc);
+    expect(fcBBox, equals([-10, 100, 4, 130]), reason: "featureCollection");
```

Comment:
I think, you cannot compare it to just an array, because `bbox()` returns a `BBox` object

Reply:
but I don't know if it actually works, it could because `BBox` extends `CoordinateType`, which itself implements `Iterable`.

Reply:
👍